### ISSUE TITLE
Divide by zero

### ISSUE THREAD

**Issue by niekvlessert** [emscripten-core/emscripten#9966]
I'm aware this is not a very precise question but it's the best I can do.

The issue I'm having is that my c code that compiled & ran just fine a year ago now doesn't work anymore on the latest Emscripten, I get those divide by zero errors. The code opens a chiptune music file (zipped or not) and tries to parse it. When I compile it on the command line it works fine. I compile a chiptune music file in the JS file, and I'm sure it can be opened, I can display the composer and stuff from the file. However somehow the values of variables are different when using it on the command line, I checked using printf. I would say the general idea is that it shouldn't differ... ;)

So basically my question is what can cause this difference in behaviour? I did -O0. I have no clue on where to start to debug if values are different. I can provide whatever you need from my little project.

**Comment by niekvlessert** [comment]
Excuse me, my own fault...
